timelock-server/src/main/java/com/palantir/atlasdb/timelock/TimeLockServerLauncher.java

```diff
@@ -73,18 +68,13 @@ public void initialize(Bootstrap<CombinedTimeLockServerConfiguration> bootstrap)
         MetricRegistry metricRegistry = SharedMetricRegistries.getOrCreate(
                 "AtlasDbTest" + UUID.randomUUID().toString());
         bootstrap.setMetricRegistry(metricRegistry);
-        bootstrap.setObjectMapper(ObjectMappers.newServerObjectMapper());
-
-        bootstrap.getObjectMapper().registerSubtypes(NonBlockingFileAppenderFactory.class);
-        bootstrap.getObjectMapper().setSubtypeResolver(new DiscoverableSubtypeResolver());
-        bootstrap.getObjectMapper().registerModule(new Jdk8Module());
+        bootstrap.setObjectMapper(ObjectMappers.json());
         super.initialize(bootstrap);
     }
 
     @Override
     public void run(CombinedTimeLockServerConfiguration configuration, Environment environment)
             throws JsonProcessingException {
-        environment.getObjectMapper().registerModule(new Jdk8Module()).registerModule(new JavaTimeModule());
```

Comment:
Do we know why we used to register the module here instead of in _initialize_ method? Because now we are moving this line to there.

Reply:
I don't think there's any particular reason - just a historical artifact. There's no reason why we can't register these in` initialize`.